Update README.md (#500)
hashicorp/vscode-terraform: README.md

* Multi-folder workspaces are not yet supported. ([info](https://github.com/hashicorp/vscode-terraform/issues/329#issuecomment-639378948))
* A number of different folder configurations (specifically when your root module is not a parent to any submodules) are not yet supported. ([info](https://github.com/hashicorp/terraform-ls/issues/32#issuecomment-649707345))

## User Group

> We have a new user group for the Terraform VS Code extension. This group offers users of the Terraform VSCode Extension with a place to provide feedback, access to pre-release versions, and opportunities to influence the direction of this project. Collaboration and updates will be shared via HashiCorp Discuss (discuss.hashicorp.com), along with occasional live events.

Reviewer: Suggestion to differentiate a new group from a group for new users.
```suggestion
We're starting a user group for the Terraform VS Code extension. This group offers users of the Terraform VSCode Extension with a place to provide feedback, access to pre-release versions, and opportunities to influence the direction of this project. Collaboration and updates will be shared via HashiCorp Discuss (discuss.hashicorp.com), along with occasional live events.
```
